packages/device_info/device_info/example/pubspec.yaml

```diff
@@ -4,14 +4,13 @@ description: Demonstrates how to use the device_info plugin.
 dependencies:
   flutter:
     sdk: flutter
+  e2e:
```

Comment:
Why move this to a normal dependency? Please leave it as a dev_dependency, and fix the path: `../../../e2e`?

Reply:
It was giving tests error when I made a PR last time. I'll reverse the change right now.

Reply:
Thanks! I think I've seen the errors that you're describing, and they mean that you need to add e2e as a dev_dependency on the core plugin, not only on the example (I think it needs to be on both :P)

Let's confirm with the CI!

Reply:
I took the example from the url_launcher plugin and it didn't had e2e in the core plugin's pubspec.yaml. e2e was only present in the example. Lets confirm with the CI.